### ISSUE TITLE
Invalid value for --max-columns

### ISSUE THREAD

**Issue by fheinicke** [Gruntfuggly/todo-tree#137]
Seems like a change in the recent 0.0.123 update where the default value for max-columns was introduced leads to an error when loading the extension.

Debug log:
```
Searching c:\Workspace\MyProject...
Command: "c:\Users\fheinicke\AppData\Local\Programs\Microsoft VS Code\resources\app\node_modules.asar.unpacked\vscode-ripgrep\bin\rg.exe" --no-messages --vimgrep -H --column --line-number --color never --max-columns=1000-i  -e "((//|#|<!--|;|/\*|^)\s*(TODO|FIXME|HACK)\b|^\s*- \[ \])" "c:\Workspace\MyProject"
error: Invalid value for '--max-columns <NUM>': invalid digit found in string
```

The missing whitespace after `--max-columns=1000` seems to be the issue. Manually settings the ripgrep args to include a whitespace at the end makes the extension functional again.

Adding a whitespace to non-empty options.additional before adding the `-i` argument should be sufficient?
https://github.com/Gruntfuggly/todo-tree/blob/74d69faff1f5311027e60e161a3d297f11104299/extension.js#L244

User settings:

```json
"todo-tree.defaultHighlight": {
    "type": "tag",
    "foreground": "green",
    "opacity": 0
},
"todo-tree.customHighlight": {
    "TODO": {
        "icon": "pin",
        "foreground": "#67c651",
    },
    "FIXME": {
        "icon": "tools",
        "foreground": "#c65151"
    },
    "HACK": {
        "icon": "issue-opened",
        "foreground": "#e2e056"
    }
},
"todo-tree.tags": [
    "TODO",
    "FIXME",
    "HACK"
],
"todo-tree.regexCaseSensitive": false,
"todo-tree.showInExplorer": false,
"todo-tree.expanded": true,
"todo-tree.debug": true,
"todo-tree.excludeGlobs": [
    "**/node_modules/**"
],
```

**Comment by Gruntfuggly** [comment]
Good spot.  I probably need to process the options more cleverly...

**Comment by Gruntfuggly** [comment]
Please try 0.0.124.

**Comment by fheinicke** [comment]
Can confirm that it is fixed in 0.0.124. Thanks once again for your work, @Gruntfuggly !
